docs/_docs/01_getting-started/install.md

```diff
@@ -11,7 +11,12 @@ nav_title_link: /docs/
 
 ## Install Terragrunt
 
-Note that third-party Terragrunt packages may not be updated with the latest version, but are often close. Please check your version against the latest available on the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases).
+### Download from releases page
+
+You can install Terragrunt manually by going to the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases), downloading the binary for your OS, renaming it to `terragrunt`, and adding it to your PATH.
+
+Note that all the different package managers are third party.The third party Terragrunt packages may not be updated with the latest version, but are often close. Please check your version against the latest available on the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases).
+The recommended installation option is via the [Releases Page](https://github.com/gruntwork-io/terragrunt/releases).
```

Comment:
It would be easy to miss this in its current location. How about reformatting as follows:

```md
### Download from the releases page 

(content you have now)

### Install via a package manager

(disclaimer you have above)

* **Windows**: (instructions for Windows)

* **macOS**: (instructions for Mac)

(etc)
```

Reply:
I have the changes.